cypress/components/dialog/dialog.cy.tsx

```diff
@@ -101,8 +102,7 @@ context("Testing Dialog component", () => {
     it("should render close icon when ShowCloseIcon is set to true. When you click the CloseIcon the Dialog is closed", () => {
       CypressMountWithProviders(<stories.DialogComponent />);
       dialogPreview().should("exist");
-      closeIconButton().click();
-      cy.wait(1000);
+      closeIconButton().click().as("waitForClick");
```

Comment:
we don't use this

we can do:
`cy.wait("@waitForClick")`

and only then this will be a case of using an alias